web-client/public/js/grnsight.js

```diff
@@ -12,6 +12,5 @@ import { setupHandlers } from "./setup-handlers";
 
 setupHandlers(grnState);
 updateApp(grnState);
-
```

Comment:
_Almost_ got this restored to the original 🙂 You missed a deleted blank line that shouldn't have been deleted. Remember, when undoing changes, the undo isn't complete until _the file itself disappears from the PR's list of changed files_.